### ISSUE TITLE
admin notice even if hosts editable

### ISSUE THREAD

**Issue by oskude** [cogitatio/vagrant-hostsupdater#97]
just a minor nag,

my `/etc/hosts` is editable by my user

```
❱ id
uid=1000(osku) gid=1000(osku) …
❱ ls -l /etc/hosts 
-rw-rw-r-- 1 root osku 79 Feb 11 15:27 /etc/hosts
❱ cat /etc/hosts
127.0.0.1   localhost.localdomain   localhost
::1     localhost.localdomain   localhost
```

yet `vagrant-hostsupdater` says i need administrator rights

```
❱ vagrant up
…
==> default: [vagrant-hostsupdater] Checking for host entries
==> default: [vagrant-hostsupdater] Writing the following entries to (/etc/hosts)
==> default: [vagrant-hostsupdater]   10.20.30.40  foo.bar  # VAGRANT: 35c0ed0c6c424841ee2225fb94b15931 (default) / 335af14e-c6b5-4a82-ad70-c7f2196b2a9c
==> default: [vagrant-hostsupdater] This operation requires administrative access. You may skip it by manually adding equivalent entries to the hosts file.
…
```

but does actually modify `/etc/hosts`

```
❱ cat /etc/hosts 
127.0.0.1   localhost.localdomain   localhost
::1     localhost.localdomain   localhost

10.20.30.40  foo.bar  # VAGRANT: 35c0ed0c6c424841ee2225fb94b15931 (default) / 335af14e-c6b5-4a82-ad70-c7f2196b2a9c
```

**Comment by cgsmith** [comment]
Good point @oskude - The message was intended to let the user know that if it was not added that they could add it manually.  It can be changed to only show if the file is not written to.

**Comment by cbirajdar** [comment]
Should be addressed by #130!
